src/stavisky_lab_to_nwb/simulated_data/__init__.py

```diff
@@ -1,2 +1,6 @@
-from .simulated_databehaviorinterface import SimulatedDataBehaviorInterface
-from .simulated_datanwbconverter import SimulatedDataNWBConverter
+# from .simulated_databehaviorinterface import StaviskyPhonemeLogitsInterface, StaviskyDecodedTextInterface
+# from .simulated_dataecephysinterface import StaviskySpikingBandPowerInterface
+# from .simulated_datasortinginterface import RedisSortingInterface
+# from .simulated_datarecordinginterface import RedisRecordingInterface
+from .simulated_datatrialsinterface import StaviskyTrialsInterface
+from .simulated_datanwbconverter import StaviskyNWBConverter
```

Comment:
One little thing - while the names of the classes have been changed away from 'simulation', the word still appears in the submodule (folder) and file names

Reply:
Gotcha, would `stavisky` or `card` be a reasonable conversion project name, or would something more descriptive be better?